### PR TITLE
Change TFM to netcoreapp2.0

### PR DESCRIPTION
**Pull request by pranavkm**

### CODE REVIEW

File: src/Microsoft.AspNetCore.Diagnostics/baseline.netfranework.json [hunk 1]
```diff
@@ -1,2213 +0,0 @@
-{
```
Comment (Eilon) [inline src/Microsoft.AspNetCore.Diagnostics/baseline.netfranework.json]: Did you notice that the filename of this file is wrong? What's up with that? cc @javiercn

Comment (Eilon) [inline src/Microsoft.AspNetCore.Diagnostics/baseline.netfranework.json]: (Not that it matters anymore; it's just weird.)

**Comment by Eilon** [comment]
@ajcvickers - would you like someone on your team to also give this a quick once-over for the EF database error diagnostics page changes?

File: src/Microsoft.AspNetCore.Diagnostics/baseline.netfranework.json (hunk 1 above)
Comment (pranavkm) [inline src/Microsoft.AspNetCore.Diagnostics/baseline.netfranework.json]: Yeah. There was another baseline.netframework.json next to it too.

**Comment by ajcvickers** [comment]
@Eilon Sorry I didn't get to this sooner. It looks good to me.

**Pull request merged**